src/modules/awake/Awake/NativeMethods.txt

```diff
@@ -0,0 +1,22 @@
+AllocConsole
+CreateFile
+CreatePopupMenu
+DestroyMenu
+FindWindowEx
+GetClassName
+GetCurrentThreadId
+GetPwrCapabilities
+GetWindowThreadProcessId
+HMENU
+InsertMenu
+PostQuitMessage
+SendMessage
+SetConsoleCtrlHandler
+SetForegroundWindow
+SetStdHandle
+SetThreadExecutionState
+TrackPopupMenuEx
+WM_CLOSE
+WM_COMMAND
+WM_GETTEXT
+WM_USER
```

Comment:
Why is NativeMethods.txt needed?

Reply:
This is required by https://github.com/microsoft/CsWin32, which is a build-time dependency to generate PInvoke calls instead of declaring them manually. I'll replace `NativeMethods` modules in other toys later. In Awake it also helped resolved HMENU issue.

Reply:
cool